generators/k8helmpipeline/README.md

```diff
@@ -0,0 +1,41 @@
+# K8helmpipeline
+
+The K8helmpipeline sub-generator creates a CI/CD pipeline in Visual Studio Teams Services (VSTS) and generates the template files needed to provision a basic Kubernetes web application using helm.
+
+## Installing
+
+Please refer to the [home](https://github.com/DarqueWarrior/generator-team) repository to install the Yeoman generator 
+
+
+
+
+## Prerequisites

```

Comment:
I recommend asking another person to run through your instructions and see how easily it is to follow. You may want to just provide the commands here to create these artifacts oppose to linking the user to more docs to perform the creation.